tests/test_entrypoint.py

```diff
@@ -167,6 +168,21 @@ def test_generation_cargo_lock(self):
         )
 
 
+class TestDockerImageSBOMGeneration(unittest.TestCase):
+    def test_generation_docker_image(self):
+        """
+        Test CycloneDX generation of SBOM
+        from a Docker Image.
+        """
+
+        output_file = "test_docker_image_generation.json"
+
+        generation_return_code = run_trivy_docker_image(
+            docker_image="nginx:latest",
+            output_file=output_file,
+        )
+
```

Comment:
No `assert` to check if the return code is the one expected? Though if the return code is non-zero, the output from trivy command probably wouldn't be json either and the test would raise and error. Still without assert the test case doesn't seem like a test case :-)